applications/osb-portal/src/components/header/Banner.tsx

```diff
@@ -39,7 +39,7 @@ export const Banner = (props: any) => {
   const text1 =
     user === null
       ? "Welcome to the Open Source Brain v2.0 beta release!"
-      : `Welcome back ${user.firstName}`;
+      : `Welcome back, ${user.firstName}`;
```

Comment:
Do we still need the Banner?

Reply:
no, the edit was by mistake, I will remove all unnecessary files after making sure that I have completed everything without forgetting anything